stories/Breakpoints/Breakpoints.story.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Heading,
+  CodeExample,
+  Title,
+  StoryContainer,
+} from '@catho/quantum-storybook-ui';
+import { storiesOf } from '@storybook/react';
+import { BREAKPOINTS } from '../../components/shared';
+import { query } from '../../components/Grid/sub-components/shared';
+
+const mediaQueries = query(BREAKPOINTS);
+
+const StyledDivExample = styled.div`
+  padding: 15px 20px;
+  color: white;
+  background-color: red;
+
+  ${mediaQueries.small`
+    background-color: blue;
+  `}
+
+  ${mediaQueries.medium`
+    background-color: green;
+  `}
+
+  ${mediaQueries.large`
+    background-color: black;
+  `}
+
+  ${mediaQueries.xlarge`
+    background-color: orange;
+  `}
+`;
+
+storiesOf('Foundation', module).add('Breakpoints', () => (
+  <>
+    <Heading title="Breakpoints">
+      Breakpoints - Quantum provides a breakpoints object that you can use with
+      styled components.
+      <br />
+      You can create different style rules following the Quantum breakpoints.
+    </Heading>
+
+    <StoryContainer>
+      <Title as="h2">How to use</Title>
+
+      <p>
+        First import `BREAKPOINTS` from `shared`. Second import `query` function
```

Comment:
Where there are the crase, how about using a `<code>` tag?
Only to keep the pattern.

ex:
now:
 ```First import `BREAKPOINTS` from `shared`.```
changed:
```First import <code>BREAKPOINTS</code> from <code>shared</code>.```


Resut

Before:
![Captura de tela de 2019-11-27 11-12-37](https://user-images.githubusercontent.com/3389749/69730444-d89f0e80-1106-11ea-98b1-a74cc8622e3e.png)

after:
![Captura de tela de 2019-11-27 11-12-14](https://user-images.githubusercontent.com/3389749/69730425-cb821f80-1106-11ea-94de-6e4e8ea43972.png)

Reply:
Done